co-log/src/Colog/Message.hs

```diff
@@ -283,10 +293,7 @@ __Examples:__
 See 'fmtMessage' if you don't need both time and thread id.
 -}
 fmtRichMessageDefault :: MonadIO m => RichMessage m -> m Text
-fmtRichMessageDefault RichMessage{..} = do
-    maybeThreadId  <- extractField $ TM.lookup @"threadId"  richMessageMap
-    maybePosixTime <- extractField $ TM.lookup @"posixTime" richMessageMap
-    pure $ formatRichMessage maybeThreadId maybePosixTime richMessageMsg
+fmtRichMessageDefault msg = fmtRichMessageCustom msg formatRichMessage
```

Comment:
Thanks for trying to cleanup code duplication! I've created another issue with the discussion about possible improvements to code modularity.

* https://github.com/kowainik/co-log/issues/129

Reply:
Would you like me to work on #129 in this same PR or should we shoot for it later?

Reply:
@sphaso Much later. It's not even clearer to implement this 🤔 But if you would like to work with the advanced Haskell topics, let me know